common-test/src/main/java/io/strimzi/test/StrimziRunner.java

```diff
@@ -430,9 +430,9 @@ private Statement withClusterOperator(Annotatable element,
                     JsonNodeFactory factory = new JsonNodeFactory(false);
                     ObjectNode resources = new ObjectNode(factory);
                     ObjectNode requests = new ObjectNode(factory);
-                    requests.put("cpu", "200m").put("memory", "128Mi");
+                    requests.put("cpu", "200m").put("memory", "256Mi");
                     ObjectNode limits = new ObjectNode(factory);
-                    limits.put("cpu", "1000m").put("memory", "128Mi");
+                    limits.put("cpu", "1000m").put("memory", "256Mi");
```

Comment:
Are there changes already in the master? I saw a previous commit on master for that.

Reply:
Yes, but they were not in master when I worked on this. Only after this PR I started investigate the issue in master.